src/Modules/Render/ViewScene.cc

```diff
@@ -233,6 +245,9 @@ void ViewScene::syncMeshComponentFlags(const std::string& connectedModuleId, Mod
 
 void ViewScene::execute()
 {
+  auto state = get_state();
+  auto executionReporter = ScopedExecutionReporter(state);
```

Comment:
You got it, cool. Aren't those types neat?

Reply:
Yeah, I didn't know about it. It's very useful